_sass/components/_c-section-title.scss

```diff
@@ -1,6 +1,6 @@
 .c-section-title {
   position: relative;
-  padding-bottom: rem(50px);
+  // padding-bottom: rem(50px);
```

Comment:
Be extremely careful when doing changes in this general elements.  You might be affecting more elements.
Be sure you are not crashing another elements and if so, make and exception (&.-my-awesome-exception)